addons/stock/report/report_deliveryslip.xml

```diff
@@ -203,7 +203,7 @@
                             <img t-att-src="image_data_uri(o.signature)" style="max-height: 4cm; max-width: 8cm;"/>
                         </div>
                         <div t-if="o.partner_id" class="offset-8 text-center">
```

Comment:
Dang, I didn't realize you could have a res.partner without a name. I'd say add the check into the existing t-if in the `div` since we're not doing anything with the o.partner_id

```suggestion
                        <div t-if="o.partner_id.name" class="offset-8 text-center">
```

Reply:
We didn't realize it either! The sql constraint doesn't seem to come into affect for any `res.partner` record that isn't the `contact` type. Also doh, yes that is obviously a better change, amended.